test/unit/MailHelperTest.php

```diff
@@ -70,4 +70,64 @@ public function testEmailName()
             '{"name":"\\"O\'Keeffe, John \\\\\\"Billy\\\\\\"\\"","email":"test@example.com"}'
         );
     }
+
+    public function testA()
```

Comment:
The `testA` method name is not proper because I don't know this test method actually tests.

Can you name the proper test method name?

Reply:
@peter279k done!